pages/doc/wavefront_release_notes.md

```diff
@@ -26,21 +26,55 @@ For information about the two subscription types and how they differ, see [Subsc
  
 {% include note.html content="We will support both original and onboarded subscriptions until all original subscriptions are migrated to VMware Cloud services."%}
 
-
-## 2023-29.x Release Notes
+## 2023-31.x Release Notes
 
 ### Original and Onboarded Subscriptions
 
-<!-- Uncomment and release for 20223-30.x - next release
+* **Logs GA release**: This release marks the GA release for the [Logs feature](logging_overview.html). You can:
```

Comment:
How about something like:
With this release, we launch the General Availability of our [Logs feature](logging_overview.html). You can:

The note and the bullets need to be nested under this one. I would also move the note below the bullets.